services/QuillLMS/config/routes.rb

```diff
@@ -369,6 +369,8 @@
       get 'rule_feedback_history/:rule_uid' => 'rule_feedback_histories#rule_detail'
       get 'prompt_health/:activity_id' => 'rule_feedback_histories#prompt_health'
 
+      get 'app_settings/:user_id' => 'app_settings#index'
```

Comment:
I think we'd want this is a nested resource route added to the users resource below, e.g.
```ruby
resources :users, only: [:index] do
  resources :app_settings, only: [:index]
end